docker/esy.Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM esydev/esy:nightly-alpine-latest as builder
+
+RUN apk add libexecinfo-dev && apk --update add ca-certificates
+
+WORKDIR /app
+
+# Add things that doesn't change much, or should bust cache when it does
+COPY ./esy.lock esy.json ./
```

Comment:
This doesn't copy the `esy.lock` directory properly (causing esy to re-solve). I used

```
COPY ./esy.lock ./esy.lock
COPY ./esy.json ./esy.json
```

Reply:
Can you push this change? This is probably why we need the second `esy install`

Reply:
Actually, I can make the change

Reply:
I fixed this, removed the extra `esy install` and changed to `esy build --release` things are much better now :)